src/api/app/views/webui/request/show.html.erb

```diff
@@ -211,12 +211,16 @@
   <div class="grid_10 alpha">
     <% if @superseding %>
       <% @superseding.each do |supersed| %>
-        <div class="grid_10 box box-shadow alpha omega">
-          <h2 class="box-header"><a class="supersed_comments_link">Show outdated comments</a>for superseded <%= link_to "request #{supersed.number}", number: supersed.number %></h2>
-          <div class="superseded_comments hidden">
-            <%= render :partial => 'webui/comment/show', locals: { commentable: supersed } %>
+        <% if supersed.comments.any? %>
```

Comment:
It looks like this could be combined with the condition in line 212.

Reply:
Nope, as @superseding is a list and supersed is one element of the list...